ci/build.yml

```diff
@@ -104,16 +104,20 @@ jobs:
     - template: report-end.yml
 
 - job: Linux_scala_2_12
+  # We disable this for release builds. The actual release happens via the regular Linux builds
+  # and running tests on two scala versions doesn’t add any value if the tests already
+  # passed on the respective PRs that added the changes. The only difference here
+  # is the release version which should not affect things differently depending on the
+  # Scala version.
+  condition: and(succeeded(),
```

Comment:
@garyverhaegen-da iirc this is fine and skipping this doesn’t break jobs like collect_build_data that depend on it. Can you confirm this?

Reply:
Not sure if I can refer to the variable directly below. This seemed to match what we do in other places.

Reply:
As far as I understand, yes, you should be able to access dependencies in `condition`, and that's implicitly what things like `succeeded` do. I usually prefer to write the `condition` block after the `dependsOn` one for that reason, but I don't think Azure cares about ordering here.